### PR TITLE
Load and show creation_date of shortlink

### LINKED ISSUE #5220: [REQUEST]: show timestamp when shortlink was created
### Is your feature request related to a problem? Please describe

Suggested during cpponsea

To show when a shortlink was created (after havent clicked one somewhere and CE showing the contents)

Should probably make an API call or include it in `/api/shortlinkinfo` as well

### Describe the solution you'd like

Not sure where to place it in the UI yet

### Describe alternatives you've considered

-

### Additional context

Not applicable

### PR DESCRIPTION
**Pull request by partouf**
Fixes #5220 

Todo:
- [x] Improve types
- [x] Improve layout

### CODE REVIEW

File: static/main.ts [hunk 1]
```diff
@@ -532,6 +534,34 @@ function getDefaultLangId(subLangId: LanguageKey | undefined, options: CompilerE
...
+
+function showShortlinkInfoButton() {
+    const div = $('.shortlinkInfo');
+    div.show();
+}
+
+function initShortlinkInfoButton() {
+    const button = $('.shortlinkInfo div');
+    if (options.metadata && options.metadata['ogCreated']) {
+        button.popover({
```
Comment (RubenRBS) [inline static/main.ts:550]: If I understand the code correctly (Can't run locally right now), this needs the user to click on the button to fnid the creation time, and is not shown n the button itself, right?

Oh I guess this way we can add more info in the future now that I think about it! Nice!

Comment (partouf) [reply static/main.ts:550]: That was the idea yeah, but also tried alt text. Maybe it can still be plain text, will experiment.

**Comment by partouf** [comment]
Ok, I think this is a slightly better solution now:

Text visible when enough room:
Screenshot from 2023-07-06 23-48-11

Hidden when not enough room:
Screenshot from 2023-07-06 23-48-35

When clicked on info icon:
Screenshot from 2023-07-06 23-49-00

**Comment by partouf** [comment]
(using manual nonsense of formatting datetime because I didn't want to add moment.js just for this one thing)

**Comment by partouf** [comment]
now live

**Comment by partouf** [comment]
(in the end had to limit it to just an icon and not the text, did not test locally with MOTD and it looked rubbish)

**Pull request merged**